rero_mef/concepts/rero/tasks.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this file is required now ? All the code content are comment.

Reply:
+1